new-site/src/docs/elements/components/notification/code.mdx

```diff
@@ -0,0 +1,215 @@
+---
+slug: '/elements/components/notification/code'
+title: 'Notification - Code'
+---
+
+import { Notification } from 'hds-react';
+import { Link } from 'hds-react';
+
+## Code
+
+### Code examples 
+
+#### Inline
+<Playground>
+
+
+```jsx
+<>
+<Notification label="New messages">You have received new messages.</Notification>
+<Notification label="Form done" type="success" style={{marginTop: 'var(--spacing-s)'}}>Form submit was successful!</Notification>
+<Notification label="Slow loading" type="alert" style={{marginTop: 'var(--spacing-s)'}}>Loading is taking longer than expected.</Notification>
+<Notification label="Missing information" type="error" style={{marginTop: 'var(--spacing-s)'}}>Form is missing critical information.</Notification>
+</>
+```
+
+```html
+<div>
+<section aria-label="Notification" class="hds-notification">
+  <div class="hds-notification__content">
+    <div class="hds-notification__label" role="heading" aria-level="2">
+      <span class="hds-icon hds-icon--info-circle-fill" aria-hidden="true"></span>
+      <span>New messages</span>
+    </div>
+    <div class="hds-notification__body">You have received new messages.</div>
+  </div>
+</section>
+<br />
```

Comment:
The linebreaks seems to be little different in React tab. Either use `<br />` in both, or then the styling in both (section): `style={{marginTop: 'var(--spacing-s)'}}`

Reply:
Which one would be better? In the future, we need to remember it is now much easier to edit and copy the source code. So if there are some parts of the source that are going to be annoying for the developer, we should edit them.

Reply:
Both look good to me, you can decide.